### ISSUE TITLE
FilePathField

### ISSUE THREAD

**Issue by biggosh** [tomchristie/django-rest-framework#3520]
I tried to use this field in the model and rest framework always raises an error:

__init__() missing 1 required positional argument: 'path'

Is this field type supported?

Thanks

**Comment by jpadilla** [comment]
@biggosh could you perhaps share what your model and serializer look like?

**Comment by biggosh** [comment]
Sorry for the delay. This is the the serializer and the original model.
Thanks

class TemplatesSerializer(serializers.ModelSerializer):
    class Meta:
        model = Templates

class Templates(models.Model):
    # file_path = models.CharField(max_length=1000, null=False, blank=False)
    file_path = models.FilePathField(path="tmp")
    title = models.CharField(max_length=100, null=False, blank=False)
    description = models.TextField(null=True, blank=True)
    code = models.CharField(max_length=10, null=False, blank=False)

    owner = models.ForeignKey('auth.User', related_name='templates')

    def __str__(self):
        return self.title

**Comment by jpadilla** [comment]
@biggosh we're probably not passing along the model field's definition along to the mapped serializer field. Could you try explicitly setting the field in your serializer like so:

```python
class TemplatesSerializer(serializers.ModelSerializer):
    file_path = serializers.FilePathField(path='tmp')
    
    class Meta:
        model = Templates
```

**Comment by biggosh** [comment]
Ok, now it's working.

thanks

**Comment by tomchristie** [comment]
> we're probably not passing along the model field's definition along to the mapped serializer field.

Although there's a work around it appears that this issue isn't resolved until we *do* deal with the above, right?

**Comment by jpadilla** [comment]
@tomchristie sure thing. I'll get to it.

**Comment by jpadilla** [comment]
@tomchristie should we map all of the field's options: `path`, `match`, `recursive`, `allow_folders`, and `max_length`?

**Comment by tomchristie** [comment]
I guess whichever we support, yeah.

**Comment by jpadilla** [comment]
Closed by #3536
